### PR TITLE
redo (ticdc): adjust s3 timeout to 15s

### LINKED ISSUE #6931: redo changefeed in error status when using minio as redo log storage
### What did you do?

1. Start redo log changefeed, and specify minio as redo log storage target
2. Run workload
3. Check changefeed status

### What did you expect to see?

changefeed status should be normal

### What did you see instead?

changefeed status is error

```
[
  {
    "id": "redo-apply-basic-real-time-sync-s3-partition",
    "summary": {
      "state": "error",
      "tso": 435642959197110284,
      "checkpoint": "2022-08-30 07:53:25.238",
      "error": {
        "addr": "upstream-ticdc-0.upstream-ticdc-peer.cdc-fb-pbzd2.svc:8301",
        "code": "CDC:ErrOwnerUnknown",
        "message": "[CDC:ErrS3StorageAPI]s3 storage api: RequestCanceled: request context canceled\ncaused by: context deadline exceeded"
      }
    }
  }
]

```

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
/ # /tidb-server -V
Release Version: v6.1.1
Edition: Community
Git Commit Hash: 5263a0abda61f102122735049fd0dfadc7b7f8b2
Git Branch: heads/refs/tags/v6.1.1
UTC Build Time: 2022-08-25 10:42:41
GoVersion: go1.18.5
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false

```

Upstream TiKV version (execute `tikv-server --version`):

```console
/ # /tikv-server -V
TiKV 
Release Version:   6.1.0
Edition:           Community
Git Commit Hash:   a116fcae5fc7b80d17906dedf2bb4d3929fb4dcc
Git Commit Branch: heads/refs/tags/v6.1.1
UTC Build Time:    2022-08-23 08:28:10
Rust Version:      rustc 1.60.0-nightly (1e12aef3f 2022-02-13)
Enable Features:   jemalloc mem-profiling portable sse test-engine-kv-rocksdb test-engine-raft-raft-engine cloud-aws cloud-gcp cloud-azure
Profile:           dist_release

```

TiCDC version (execute `cdc version`):

```console
bash-5.1# /cdc version
Release Version: v6.1.1
Git Commit Hash: 8201ed98668e940c7a9051bbd4ec09a0e2a9aa15
Git Branch: heads/refs/tags/v6.1.1
UTC Build Time: 2022-08-26 06:33:50
Go Version: go version go1.18.5 linux/amd64
Failpoint Build: false

```

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #6931 

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: cdc/redo/writer/file.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ const (
 	// It should be a multiple of the minimum sector size so that log can safely
 	// distinguish between torn writes and ordinary data corruption.
 	pageBytes        = 8 * common.MinSectorSize
-	defaultS3Timeout = 3 * time.Second
+	defaultS3Timeout = 15 * time.Second
```
[the remark this answers is not in the document]
Comment (asddongmen) [reply cdc/redo/writer/file.go:49]: The EE team use pingcap internal minio server to run the tests and always got timeout when it's 3s, so this PR just try to adjust it a larger value experiment.

**Review by nongfushanquan — APPROVED** [review]
when the minio system is busy , 3 s is not enough to get the latest status after writing a file .but in the s3 case , it's not a problem. so this change is just for the test case .

**Review by zhaoxinyu — APPROVED** [review]
Amazon S3 provides strong read-after-write consistency for PUT and DELETE requests of objects in your Amazon S3 bucket in all AWS Regions. We can add some comments here. @asddongmen 
Ref: https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html#ConsistencyModel

**Pull request merged**